### DISCUSSION TITLE
It is possible to see the t-states between breaks/steps?

### DISCUSSION THREAD

**Discussion by remy** [maziac/DeZog#60] [Q&A] [answered]
General question - I thought I spotted it once but might have imagined it.

I want to run my program, and when I hit a breakpoint, I want to know how many t-states it took to get there. Is that a thing that dezog does or could even do?

**Comment by maziac** [comment, marked as answer]
This feature depends on the Remote, i.e. the emulator. 
ZEsarUX and the internal simulator do this:
![Bildschirmfoto 2021-08-09 um 00 06 02](https://user-images.githubusercontent.com/3081973/128647084-52c7759a-d109-4412-a8f1-86b6a8a7b2df.jpg)

But Cspect does not.

You can look up the different capabilities in Usage.md in chapter "Remote capabilities".

**Reply by remy** [reply]
Got it - in the debug console - I knew I'd spotted it somewhere before 👍 (the build console/terminal always takes focus for me, so the debug console hides all the time - just needed to tweak my tasks.json to stop that).
